core/src/rustic_ai/core/messaging/core/message.py

```diff
@@ -137,7 +138,9 @@ def transform(
 class PayloadTransformer(Transformer):
 
     output_format: Optional[str] = Field(default=MessageConstants.RAW_JSON_FORMAT)
+    expression_type: Optional[str] = Field(default="jsonata")
     expression: Optional[str] = Field(default=None)
+    functions: Optional[Dict[str, Callable]] = {}
```

Comment:
We should be using the `expression` field and not introduce another `functions` field. The reason being that the payload transformer should be in a format that can be provided using json/yaml and cannot expect python code to be passed.